cpp/src/ray/runtime/abstract_ray_runtime.cc

```diff
@@ -389,7 +389,9 @@ std::string AbstractRayRuntime::DeserializeAndRegisterActorHandle(
     const std::string &serialized_actor_handle) {
```

Comment:
So we fix the reference cycle of a <-> a but what about a <-> b?

Reply:
I don't think we've ever supported GC for cycles with strong refs. But if they both got the actors by name, then it should work. Will add a test.